html-css/week1/Vadim/OrderProject/files/css/styles.css

```
@@ -0,0 +1,405 @@
+body {
+  margin: 0 auto;
+  font-family: Arial;
+}
+
+body a {
+  cursor: pointer;
+}
+
+/* --- Header --- */
+.order {
+  color: white;
+  display: flex;
+  flex-direction: column;
+  background: linear-gradient(92.74deg, #FD7B46 1.15%, #FE4E9B 98.48%);
+  padding: 2%;
+}
+
+.order__summary {
+  align-self: center;
+}
+
+.order__total {
+  display: flex;
+  flex-direction: column;
+}
+
+.order__totalpermonth {
+  display: flex;
+  justify-content: space-between;
+  opacity: 0.8;
+}
+
+.order__totalnow {
+  display: flex;
+  justify-content: space-between;
+  font-weight: 700;
+}
+
+.order__line {
+  border: 0;
+  clear: both;
+  display: block;
+  background: white;
+  height: 2px;
+  width: 99%;
+  margin: -6px 0 -6px 0;
+}
+
+
+.payment {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
```

Sometimes you can achieve the same result with less code.
```suggestion
  text-align: center;
```

---

noted